### DISCUSSION TITLE
blog/2024-07-18-understanding-env-var-in-nextjs

### DISCUSSION THREAD

**Discussion by giscus** [mazipan/blog-comments#22] [Q&A]
# blog/2024-07-18-understanding-env-var-in-nextjs

Bagaimana seharusnya menggunakan environment variable di Next.js

https://www.mazipan.space/blog/2024-07-18-understanding-env-var-in-nextjs

**Comment by mazipan** [comment]
💬 Leave your comment, if you have another tips and tricks, or when you have a different point of view about the topic.

Use polite sentences, do not harm anyone by using more aggressive words. 🙏

It is okay to have different perspectives in certain area.
